qiita_db/test/test_util.py

```diff
@@ -421,6 +422,68 @@ def test_purge_filepaths(self):
         self.assertTrue(exists(fp1))
         self.assertFalse(exists(fp_exp_id))
 
+    def test_purge_filepaths_null_cols(self):
+        """Tests the nulls vs not in issue in purge_filepaths"""
```

Comment:
Ahhh, now I understand the phrasing of the docstring. Thanks!

On (Feb-13-15|13:07), josenavas wrote:
> > @@ -421,6 +422,68 @@ def test_purge_filepaths(self):
> >          self.assertTrue(exists(fp1))
> >          self.assertFalse(exists(fp_exp_id))
> >  
> > +    def test_purge_filepaths_null_cols(self):
> > +        """Tests the nulls vs not in issue in purge_filepaths"""
> 
> There is an issue using the SQL command `NOT IN` if the list contains null values. There is a link in line 428 with more information. It was for clarity, but I think I did not achieve that; so I'm going to remove the docstring and leave only the comment with the link
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/837/files#r24696798